src/schema/v2/homeView/__tests__/HomeView.test.ts

```diff
@@ -4,7 +4,7 @@ import { ResolverContext } from "types/graphql"
 
 jest.mock("lib/featureFlags", () => ({
   isFeatureFlagEnabled: jest.fn((flag: string) => {
-    return !!flag.match("enable-home-view-section")
+    return ["diamond_home-view-marketing-collection-categories"].includes(flag)
```

Comment:
this was some leftover, changed it to cover the feature flag we want to cover in this file